weatherreport/SensorStubStormyRain.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+#include "IWeatherSensor.h"
+
+class SensorStubStormyRain : public IWeatherSensor {
```

Comment:
You could also use a single stub class (accepting the values in the constructor) and make each stub an instance of that class. Then we avoid creating a new class + file for each situation